src/App/Pages/Accounts/LoginPage.xaml.cs

```diff
@@ -26,9 +26,9 @@ public LoginPage(string email = null, AppOptions appOptions = null)
             _vm.Page = this;
             _vm.StartTwoFactorAction = () => Device.BeginInvokeOnMainThread(async () => await StartTwoFactorAsync());
             _vm.LogInSuccessAction = () => Device.BeginInvokeOnMainThread(async () => await LogInSuccessAsync());
+            _vm.LogInWithDeviceAction = async () => await StartLoginWithDeviceAsync();
```

Comment:
No need to add `async/await`, just do `FireAndForget()` or invoke on main thread if needed